flake8_pytest_style/errors.py

```diff
@@ -46,3 +46,8 @@ class PatchWithLambda(Error):
 class UnittestAssertion(Error):
     code = 'PT009'
     message = "use a regular assert instead of unittest-style '{assertion}'"
+
+
+class NoBareRaises(Error):
+    code = 'PT010'
+    message = 'set the match parameter in pytest.raises'
```

Comment:
I suggest we add the exception name here, i.e.:
`message = 'set the match parameter in pytest.raises({exception})'`